src/Interpreters/PeriodicLog.cpp

```diff
@@ -1,6 +1,7 @@
 #include <Interpreters/PeriodicLog.h>
 #include <Interpreters/ErrorLog.h>
 #include <Interpreters/MetricLog.h>
+#include "Functions/DateTimeTransforms.h"
```

Comment:
This include is not needed. By the looks of it, seems it was automatically added by VSCode or some other LSP?

Reply:
Yes, is is automatically added. I missed that, i will remove it, thanks.